packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -38,6 +40,11 @@ type Options = [
      */
     requireDefaultForNonUnion?: boolean;
 
+    /**
+     * Regular expression to evaluate comments to mean default case
+     */
+    defalutCaseCommentPattern?: string;
```

Comment:
[Docs] Typo, and also this should have an entry in `packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx`.

```suggestion
    defaultCaseCommentPattern?: string;
```